apiserver/service/minio_server.go

```diff
@@ -749,6 +790,7 @@ func RegisterMinIOAPI(group *gin.RouterGroup, conf gqlconfig.ServerConfig) {
 		group.DELETE("/model/files", auth.AuthInterceptor(conf.EnableOIDC, oidc.Verifier, "delete", "models"), api.DeleteFiles)
 		group.GET("/model/files/stat", auth.AuthInterceptor(conf.EnableOIDC, oidc.Verifier, "get", "models"), api.StatFile)
 		group.GET("/model/files/download", auth.AuthInterceptor(conf.EnableOIDC, oidc.Verifier, "get", "models"), api.Download)
+		group.GET("/model/files/downloadlink", auth.AuthInterceptor(conf.EnableOIDC, oidc.Verifier, "get", "models"), api.GetDownloadLink)
```

Comment:
@0xff-dev do we think about RBAC check for APIs? I think we just validate OIDC token for now, right?

Reply:
`auth.AuthInterceptor` function is to verify that the current user has `get access` to the model resource based on the token.